src/components/templates/ScreenShare/agoraHooks/useAgoraCamera.tsx

```diff
@@ -0,0 +1,88 @@
+import { useState, useEffect, useCallback } from "react";
+import AgoraRTC, {
+  IAgoraRTCClient,
+  ILocalAudioTrack,
+  ILocalVideoTrack,
+} from "agora-rtc-sdk-ng";
+
+export default function useAgoraCamera(
```

Comment:
We tend to not use default exports.

Reply:
We also don't tend to use `function`, preferring `export const foo = () => {` syntax for functions.

Reply:
We also have a `ReactHook` type.